client/src/components/Home.js

```diff
@@ -11,16 +13,24 @@ const Home = () => {
       );
       setChannels(receivedChannels);
     })();
-  }, []);
+  }, [isSubscribed]);
```

Comment:
this will cause it to refetch all the videos which is slow and inefficient, we can instead make the delete button accept a callback that will remove that channel from state. the result will be instant and without an extra fetch